### PR TITLE
Added Zsh Support

### PR DESCRIPTION
**Pull request by Pedro-Roque**
Support for Zsh users.

### CODE REVIEW

File: scripts/configure.sh [hunk 1]
```diff
@@ -311,10 +311,17 @@ if [ $native_build == 1 ] ; then
     cmake_astrobee_path=`catkin locate -s`/cmake
     grep -qF ${cmake_astrobee_path} ~/.bashrc || {
       echo -e '\n' >> ~/.bashrc \ 
-      echo 'if [[ ":$CMAKE_PREFIX_PATH:" != *":'${cmake_astrobee_path}':"* ]]; then CMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH:+"$CMAKE_PREFIX_PATH:"}'${cmake_astrobee_path}'"; fi' >> ~/.bashrc
+      echo 'if [[ ":$CMAKE_PbREFIX_PATH:" != *":'${cmake_astrobee_path}':"* ]]; then CMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH:+"$CMAKE_PREFIX_PATH:"}'${cmake_astrobee_path}'"; fi' >> ~/.bashrc
```
Comment (oleg-alexandrov) [inline scripts/configure.sh:314]: This line has a typo.

File: scripts/configure.sh [hunk 2]
```diff
@@ -311,10 +311,17 @@ if [ $native_build == 1 ] ; then
...
     }
     source ~/.bashrc
 
+    if [[ -z "${ZSH_VERSION}" ]]; then
+        echo "Setting .zshrc with environment variables..."
+        echo '# Astrobee environment variables' >> ~/.zshrc
+        echo 'source /opt/ros/'$ros_version'/setup.zsh' >> ~/.zshrc
+        echo 'if [[ ":$CMAKE_PbREFIX_PATH:" != *":'${cmake_astrobee_path}':"* ]]; then CMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH:+"$CMAKE_PREFIX_PATH:"}'${cmake_astrobee_path}'"; fi' >> ~/.zshrc
```
Comment (oleg-alexandrov) [inline scripts/configure.sh:322]: This line has a typo.

**Comment by Pedro-Roque** [comment]
Thanks @oleg-alexandrov , just pushed a typo fix

**Comment by oleg-alexandrov** [comment]
Thank you.

Running configure.sh will each time add more lines to .zshrc. I hope you can test if that is the case. Then note the logic used with .bashrc where a grep call first checks if the desired line is already in. Maybe you can do something similar and test that this time it does not add lines to .zshrc if already present.

File: scripts/configure.sh [hunk 3]
```diff
@@ -315,6 +315,13 @@ if [ $native_build == 1 ] ; then
     }
     source ~/.bashrc
 
+    if [[ -z "${ZSH_VERSION}" ]]; then
+        echo "Setting .zshrc with environment variables..."
+        echo '# Astrobee environment variables' >> ~/.zshrc
+        echo 'source /opt/ros/'$ros_version'/setup.zsh' >> ~/.zshrc
+        echo 'if [[ ":$CMAKE_PREFIX_PATH:" != *":'${cmake_astrobee_path}':"* ]]; then CMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH:+"$CMAKE_PREFIX_PATH:"}'${cmake_astrobee_path}'"; fi' >> ~/.zshrc
```
Comment (marinagmoreira) [inline scripts/configure.sh:322]: I second Oleg's comment, if you notice we have before those add commands:
 grep -qF ${cmake_astrobee_path} ~/.bashrc ||

**Review by marinagmoreira — CHANGES_REQUESTED** [review]

**Comment by Pedro-Roque** [comment]
Yes indeed! I'll add that check in and re-push, thanks for the feedback @oleg-alexandrov and @marinagmoreira!

**Comment by Pedro-Roque** [comment]
Updated :)

**Comment by oleg-alexandrov** [comment]
Looks good to me. Did you test that the .zshrc gets updated, and only once?

**Comment by Pedro-Roque** [comment]
Yes, it did not add anything else to my .zshrc. I also modified the check for Zsh, to only set it if the user $SHELL is Zsh

**Comment by oleg-alexandrov** [comment]
Sounds good. I will leave it up to Marina to approve and bring in the change.

**Review by marinagmoreira — APPROVED** [review]

**Comment by Pedro-Roque** [comment]
@marinagmoreira should this be merged? or waiting for @oleg-alexandrov ?

**Pull request merged**